### PR TITLE
riscv: get further in the boot process [WIP]

### PR DESCRIPTION
**Pull request by rminnich**
we have print. Now need to work out meminit.

Signed-off-by: Ronald G. Minnich rminnich@gmail.com

### CODE REVIEW

File: sys/src/9/riscv/main.c [hunk 1]
```diff
@@ -119,14 +120,34 @@ void bsp(void)
 
 	static Lock l; // to ensure initialization.
 	if (canlock(&l)) msg ("L can be locked\n"); else die("Can't lock L\n");
-	ilock(&l);
+	unlock(&l);
 	if (canlock(&l)) msg ("L can be NOT be locked OK\n"); else die("Can lock L after lock\n");
```
Comment (dancrossnyc) [inline sys/src/9/riscv/main.c]: "L can be NOT be locked OK\n" <-- I think that's got an extra 'be': how about, "L can NOT be locked OK\n"

Comment (dancrossnyc) [inline sys/src/9/riscv/main.c]: Or even, "L can NOT be locked after lock\n"

Comment (dancrossnyc) [inline sys/src/9/riscv/main.c]: (Or, "L can NOT be locked while held\n")

**Comment by rminnich** [comment]
you can kind of ignore this for now, I am using github as a storage device.

**Comment by elbing** [comment]
wow, this looks epic.

**Pull request closed without merging**